packages/asset-service/src/service/descriptions/pt.json

```
@@ -1,9 +1,10 @@
+  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa uma pool de liquidez na Curve, a corretora descentralizada líder no setor. Holders podem ganhar taxas dos usuários que negociam na pool e também podem depositar seus tokens LP na[ ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e [ETH](/assets/eip155:1/slip44:60). Para ter acesso ao LP da Curve de CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e/ou [ETH](/assets/eip155:1/slip44:60) na [Curve](https://curve.fi/crveth/deposit). Por favor, saiba que as pools são compostas por ativos de preços de diferentes e estão sujeitas a perda impermanente. ",
```

Trailing whitespace, remove 
```suggestion
  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa uma pool de liquidez na Curve, a corretora descentralizada líder no setor. Holders podem ganhar taxas dos usuários que negociam na pool e também podem depositar seus tokens LP na[ ShapeShift](/defi/earn) para obter recompensas. Esta pool de liquidez contém [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e [ETH](/assets/eip155:1/slip44:60). Para ter acesso ao LP da Curve de CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) e/ou [ETH](/assets/eip155:1/slip44:60) na [Curve](https://curve.fi/crveth/deposit). Por favor, saiba que as pools são compostas por ativos de preços de diferentes e estão sujeitas a perda impermanente.",
```